api/validators/MerchStoreRequests.ts

```diff
@@ -300,3 +320,8 @@ export class EditOrderPickupEventRequest implements IEditOrderPickupEventRequest
   @IsDefined()
   pickupEvent: OrderPickupEventEdit;
 }
+
+export class GetOrderQueryParams implements IGetOrderQueryParams {
```

Comment:
The route is also for users to view their cancelled/fulfilled/pending/missed orders to view for themselves.

Reply:
Frontend could have some pop-up like "are you really really sure". Also we should prob log picking up orders, canceling orders, and canceling events as activities.

Reply:
We will leave it in for now and wait for Steven's green light if we need to keep this route in or not since it's not 100% solidified yet based on admin designs. I've just implemented the `/order/cleanup` route, and we can easily delete this route later if we decide we don't need it

Reply:
Can we move the cleanup route to a separate PR?